Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -1298,7 +1298,9 @@ bool TestController::resetStateToConsistentValues(const TestOptions& options, Re
             return false;
         }
     }
-    
+
+    WKPageClearBackForwardListForTesting(TestController::singleton().mainWebView()->page(), nullptr, [](void*) { });
```

Comment:
If this causes any flaky tests, we may want to spin the run loop and wait for the callback like we do a few other things.  If it doesn't cause flaky tests, this is fine.